cosmosdb/management_endpoint.bal

```diff
@@ -379,14 +389,14 @@ public client class ManagementClient {
         return finalStream;
     }
 
-    # Delete an existing User Defined Function.
+    # Deletes an existing User Defined Function.
```

Comment:
Check "User Defined Function" in all the places and use a standard approach with capitalization.